pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1056,6 +1056,9 @@
     ?>  ?=(@ content.packet)
     =+  ;;  [signature=@ signed=@]  (cue content.packet)
     =+  ;;  =open-packet            (cue signed)
+    ::  assert the comet can't pretend to be a moon or other address
+    ::
+    ?>  (gth 64 (met 0 sndr.packet))
```

Comment:
Should this also just use `clan:title`? We probably don't want 129-bit "comets" either…

Reply:
unnecessary -- `sndr.packet` is read from a 128-bit slot in the packet, not from a jammed noun